app/services/crm/leadsquared/mappers/conversation_mapper.rb

```diff
@@ -0,0 +1,101 @@
+class Crm::Leadsquared::Mappers::ConversationMapper
+  include ::Rails.application.routes.url_helpers
+
+  def self.map_conversation_activity(conversation)
+    new(conversation).conversation_activity
+  end
+
+  def self.map_transcript_activity(conversation, messages = nil)
+    new(conversation, messages).transcript_activity
+  end
+
+  def initialize(conversation, messages = nil)
+    @conversation = conversation
+    @messages = messages
+  end
+
+  def conversation_activity
+    <<~NOTE.strip
+      New conversation started on #{brand_name}
+
+      Channel: #{channel_info}
+      Created: #{formatted_creation_time}
+      Conversation ID: #{conversation.display_id}
+      #{conversation_url}
+    NOTE
+  end
+
+  def transcript_activity
+    return 'No messages in conversation' if transcript_messages.empty?
+
+    <<~TRANSCRIPT.strip
+      Conversation Transcript from #{brand_name}
```

Comment:
It is better to use the account name / inbox name.

Conversation Transcript from Chatwoot vs Conversation Transcript from Paperlayer Inbox.

Also, please use i18n.

Reply:
Make sense! I'll make the change